server/dev.js

```diff
@@ -1,6 +1,6 @@
 // config env vars
 const path = require('path');
-require('dotenv').config({ path: path.resolve(__dirname, '../', '.env')});
+// require('dotenv').config({ path: path.resolve(__dirname, '../', '.env')});
```

Comment:
Maybe could delete this since it's just commented out :) Unless you're saving it for any particular reason